ios/FluentUI/Popup Menu/PopupMenuLegacyItemCell.swift

```diff
@@ -5,7 +5,7 @@
 
 import UIKit
 
-class PopupMenuItemCell: TableViewCell, PopupMenuItemTemplateCell {
+class PopupMenuLegacyItemCell: TableViewCell, PopupMenuItemTemplateCell {
```

Comment:
How about you name the new SwiftUI view "SelectableCell" (or something similar like "SelectableView") to avoid the name clash?
I think it'll make your life easier when integrating it to devmain and makes the name more generic without mentioning how it's being presented (Popup).

Reply:
The `PopupMenuItemCell` is intended  to work with `PopupMenutItem` (data source for PopupMenuController). In next iterations I will be taking dependency on it. I can also rename the entire cluster of `PopupMenu` to call it something else `SpringList`

This isn't meant as a generic `SelectableCell` that could be consumed by other components. List provides alternative methods to highlighted view if someone wants to achieve selection/disable functionality. 

What are your thoughts alternatively idea to use swift namespace to resolve naming conflicts?

Reply:
What is the approach that you are referring to when you mention "Swift Namespace"? Enclosing types in an enum or struct?

It also seems to me that if the experiment we discussed last week works well, the naming will be more generic and likely won't clash with the current types here.